ArchiSteamFarm/Core/Utilities.cs

```diff
@@ -291,16 +275,14 @@ public static class Utilities {
 		return first - (uint) -second;
 	}
 
-	internal static bool RelativeDirectoryStartsWith(string directory, params string[] prefixes) {
-		ArgumentException.ThrowIfNullOrEmpty(directory);
+	internal static void OnProgressChanged(object? sender, byte progressPercentage) {
+		const byte printEveryPercentage = 10;
 
-#pragma warning disable CA1508 // False positive, params could be null when explicitly set
-		if ((prefixes == null) || (prefixes.Length == 0)) {
-#pragma warning restore CA1508 // False positive, params could be null when explicitly set
-			throw new ArgumentNullException(nameof(prefixes));
+		if (progressPercentage % printEveryPercentage != 0) {
+			return;
 		}
 
-		return (from prefix in prefixes where directory.Length > prefix.Length let pathSeparator = directory[prefix.Length] where (pathSeparator == Path.DirectorySeparatorChar) || (pathSeparator == Path.AltDirectorySeparatorChar) select prefix).Any(prefix => directory.StartsWith(prefix, StringComparison.Ordinal));
+		ASF.ArchiLogger.LogGenericDebug($"{progressPercentage}%...");
```

Comment:
Since we can update plugins in parallel now, I'd like to see what binary this percentage is referring to.